docs/lib/graphqlapi/fragments/native_common/getting-started/common.md

```diff
@@ -0,0 +1,97 @@
+The Amplify API category provides an interface for retrieving and persisting your model data. The API category comes with default built-in support for AWS AppSync. The Amplify CLI allows you to define your API and provision a GraphQL service with CRUD operations and real-time functionality. The Amplify AWS API plugin leverages [AWS AppSync](https://aws.amazon.com/appsync/).
+
+## Goal
+To setup and configure your application with Amplify API to create a Todo persisted in the backend.
```

Comment:
Not sure I'd know what "a Todo" is without reading more. Can we generalize this for the reader? Such as:

> To setup and configure your application with Amplify API to save items in the backend.